### DISCUSSION TITLE
[insight]: Display the data for each point on the line chart.

### DISCUSSION THREAD

**Discussion by zhujiageng** [directus/directus#19357] [Feature Requests]
### Summary

Display the data for each point on the line chart.

### Basic Example

![image](https://github.com/directus/directus/assets/103102265/2bfa01e7-8390-4d2c-8e48-a299912913bd)

### Motivation

 view the data for the corresponding point without moving the mouse over it.

### Detailed Design

Add a switch in the panel settings to display the data for the corresponding points on the line chart.

### Must Have:
- a switch in the panel settings to display the data for the corresponding points on the line chart.

### Drawbacks

It only involves the UI side, and there's no need to make changes to the backend.

### Alternatives

none

### Adoption Strategy

This switch is off by default, so it has no effect on existing users and doesn't require migration.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
